src/specs/components/carousel/demo/data.js

```diff
@@ -0,0 +1,75 @@
+const publicUrl = process.env.PUBLIC_URL || '';
+const exampleLink = `${publicUrl}/example`;
+
+module.exports = {
+  items: [
+    {
+      title: 'EU Budget for the future',
+      description: 'Innovation, economy, environment and geopolitics',
+      link: {
+        link: {
+          label: 'Subscribe',
+          path: exampleLink,
+          aria_label: 'Subscribe',
+          icon_position: 'after',
+        },
+        icon: {
+          path: '/icons.svg',
+          name: 'corner-arrow',
+          size: 'xs',
+          transform: 'rotate-90',
+        },
+      },
+      image:
+        'https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg',
+      variant: 'image',
+      centered: false,
+    },
+    {
+      title: 'EU Budget for the future',
+      description: 'Innovation, economy, environment and geopolitics',
```

Comment:
we could showcase an example with longer description, to show that the design is not broken

Reply:
Added "Lorem ipsum", and longer example.. I actually had to fix the banners a bit to have them all at the same height on mobile and well aligned 👍